tests/BlazingSQLTest/EndToEndTests/loggingTest.py

```diff
@@ -0,0 +1,64 @@
+from Configuration import ExecutionMode
+from Configuration import Settings as Settings
+from pynvml import nvmlInit
+from Runner import runTest
+from Utils import Execution,  gpuMemory, init_context, skip_test
+
+queryType = "Logging Test"
+
+
+def main(dask_client, dir_data_file, bc, nRals):
+
+    start_mem = gpuMemory.capture_gpu_memory_usage()
+
+    def executionTest():
+        # Run Query ------------------------------------------------------
+
+        print("==============================")
+        print(queryType)
+        print("==============================")
+
+        queryId = "TEST_01"
+        query = """select * from bsql_logs"""
```

Comment:
Please change this query by adding a filter. After running thousands of queries, the return of this select * could be enormous.

Reply:
Fixed. Thanks.